app-devices/yubioath-flutter/autobuild/beyond

```diff
@@ -0,0 +1,28 @@
+abinfo "Installing files ..."
+
+if ab_match_arch amd64; then
+    export FLUTTER_ARCH="x64" 
+elif ab_match_arch arm64; then
+    export FLUTTER_ARCH="arm64"
+fi
+
+install -Dvm755 build/linux/${FLUTTER_ARCH}/release/bundle/authenticator -t \
+  "${PKGDIR}/usr/lib/${PKGNAME}/"
+install -Dvm755 build/linux/${FLUTTER_ARCH}/release/bundle/helper/authenticator-helper \
+  "${PKGDIR}/usr/lib/${PKGNAME}/helper/authenticator-helper"
+cp -rvn build/linux/${FLUTTER_ARCH}/release/bundle/{data,lib,helper} "${PKGDIR}/usr/lib/${PKGNAME}"
+find "${PKGDIR}"/usr/lib/yubioath-flutter/helper -name '*.so*' -exec chmod -v +x {} \;
+
+abinfo "Creating symlink to /usr/bin ..."
+install -dv "${PKGDIR}/usr/bin"
+ln -sv "/usr/lib/${PKGNAME}/authenticator" "${PKGDIR}/usr/bin/yubioath-flutter"
+
+abinfo "Installing icons and .desktop entry ..."
+install -Dvm644 "resources/icons/${app_id}.png" -t \
```

Comment:
Absolute path - source files must be prefixed with $SRCDIR

Reply:
The `-t` is really not needed, unless you are installing multiple source files to a common target